subsys/bluetooth/audio/has.c

```diff
@@ -113,6 +170,9 @@ static struct has_client {
 	struct  {
 		bool active_index;
 		bool control_point;
+#if defined(CONFIG_BT_HAS_FEATURES_NOTIFIABLE)
+		bool features;
+#endif /* CONFIG_BT_HAS_FEATURES_NOTIFIABLE */
```

Comment:
Maybe we should consider a bitfield for the 3 booleans? Possibly using the `atomic` API?

Reply:
Could you elaborate on why you think this adds value? Since they are all part of the same characteristic, if one has changed they will all be notified at all times, won't they?

Reply:
Sorry, Github isn't very good at this :D I meant to put the 3 booleans `active_index`, `control_point` and `features` into a single 8-bit value

Reply:
Indeed it may change as part of https://github.com/zephyrproject-rtos/zephyr/issues/57454